pkg/controller/controller.go

```diff
@@ -1165,3 +1159,50 @@ func (m *monitoringPackage) CreateVmiSecrets(binding *types.SyntheticBinding, se
 func (m *monitoringPackage) DeletePomPusher(binding string, helper v8outil.DeploymentHelper) error {
 	return v8omonitoring.DeletePomPusher(binding, helper)
 }
+
+// MCRegistrationSecret - the name of the secret that contains the cluster registration information
+const MCRegistrationSecret = "verrazzano-cluster-registration"
```

Comment:
In the case of the admin cluster, this secret will not exist unless someone creates a VMC for the admin cluster, which they would normally never do.  Instead the MCLocalRegistrationSecret will exist and have the same info.  See fetchClusterSecret cluster_utils.go in verrazzano-application-operator.

Reply:
I thought your change, @pfmackin was to create the registration secret in all cases, and the agent secret _only_ for managed clusters. Is that not true?